homeassistant/components/spc/alarm_control_panel.py

```diff
@@ -19,7 +19,7 @@
 from homeassistant.core import callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
-from . import DATA_API, SIGNAL_UPDATE_ALARM
+from . import DOMAIN, SIGNAL_UPDATE_ALARM
```

Comment:
DOMAIN in platforms are expected to represent the platform domain (alarm_control_panel)
```suggestion
from . import DOMAIN as SPC_DOMAIN, SIGNAL_UPDATE_ALARM
```

Reply:
Thanks! Fixed.